### DISCUSSION TITLE
N64 Port?

### DISCUSSION THREAD

**Discussion by CanadianJeff** [diasurgical/DevilutionX#4261] [Q&A]
could this thing run on N64 HW? using a flashcart?

I have a 64 drive and if someone is willing to port it over I can test it for them as I have a 64drive HW2 with USB hookup to load the ROM directly onto my flashcart

**Comment by AJenbo** [comment]
With only up to 8mb it won't be able to handle a port of the PC version as that requires about 32mb of memory (the original made use of virtual memory on system with less). You would first have to adapt the engine to use the PS1 assets or do some other form of reduction on the assets. Another big issue is that the game takes up 700mb but the N64 only had 64MB game packs. Again that might be doable by compressing and reducing the assets, but it's all a huge undertaking for a novelty port.

**Reply by qndel** [reply]
in other words, no xD

**Reply by vlolteanu** [reply]
The PS1 assets also doesn't feature topless succubi, so that's one extra reason not to do this port.

**Reply by vlolteanu** [reply]
don't*

**Reply by qndel** [reply]
> don't*

u can edit comments 😢

**Reply by vlolteanu** [reply]
> > don't*
> 
> u can edit comments cry

Stalin could edit pictures.

![image](https://user-images.githubusercontent.com/3142367/163274714-c0ec7f58-ac2d-4fb2-bb94-658755a8599a.png)

**Comment by CanadianJeff** [comment]
uhhhhhhhhhhhhhhhhh how do we close this looks like it is out of control

**Reply by vlolteanu** [reply]
Dude, relax. We (or rather I) were just joking around.
